lib/init-input.js

```diff
@@ -1,7 +1,8 @@
 /* global config, prompt */
 var base = require('@deck/base')
 var extend = require('extend-object')
-var exec = require('child_process').execSync
+var execSync = require('child_process').execSync
+var exec = require('child_process').exec
```

Comment:
it's used at line 41: `exec('git config --get remote.origin.url', {cwd: dir}, function (err, stdout, stderr) {`